bitnami/jaeger/templates/_helpers.tpl

```diff
@@ -0,0 +1,207 @@
+    - name: CASSANDRA_KEYSPACE
+      value: {{ .Values.cassandra.keyspace }}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use for the collector
+*/}}
+{{- define "jaeger.collector.serviceAccountName" -}}
+{{- if .Values.collector.serviceAccount.create -}}
+    {{ default (include "jaeger.collector.fullname" .) .Values.collector.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.collector.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use for the agent
+*/}}
+{{- define "jaeger.agent.serviceAccountName" -}}
+{{- if .Values.agent.serviceAccount.create -}}
+    {{ default (include "jaeger.agent.fullname" .) .Values.agent.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.agent.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use for the query
+*/}}
+{{- define "jaeger.query.serviceAccountName" -}}
+{{- if .Values.query.serviceAccount.create -}}
+    {{ default (include "jaeger.query.fullname" .) .Values.query.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.query.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create the name of the collector deployment
+*/}}
+{{- define "jaeger.collector.fullname" -}}
+    {{ printf "%s-collector" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Create the name of the collector deployment. This name includes 2 hyphens due to
+an issue about env vars collision with the chart name when the release name is set to just 'jaeger'
+ref. https://github.com/jaegertracing/jaeger-operator/issues/1158
+*/}}
+{{- define "jaeger.agent.fullname" -}}
+    {{ printf "%s--agent" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Create the cassandra secret name
+*/}}
+{{- define "jaeger.cassandra.secretName" -}}
+    {{- if not .Values.cassandra.enabled -}}
+        {{- .Values.externalDatabase.existingSecret -}}
+    {{- else -}}
+        {{- printf "%s-cassandra" (include "common.names.fullname" .) -}}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Create the cassandra secret key
+*/}}
+{{- define "jaeger.cassandra.secretKey" -}}
+    {{- if not .Values.cassandra.enabled -}}
+        {{- .Values.externalDatabase.existingSecretPasswordKey -}}
+    {{- else -}}
+        cassandra-password
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Create the cassandra user
+*/}}
+{{- define "jaeger.cassandra.user" -}}
+    {{- if not .Values.cassandra.enabled -}}
+        {{- .Values.externalDatabase.dbUser.user | quote -}}
+    {{- else -}}
+        {{- .Values.cassandra.dbUser.user | quote -}}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Create the cassandra password
+*/}}
+{{- define "jaeger.cassandra.password" -}}
+    {{- if not .Values.cassandra.enabled -}}
+        {{- .Values.externalDatabase.dbUser.password | quote -}}
+    {{- else -}}
+        {{ .Values.cassandra.dbUser.password | quote -}}
+    {{- end -}}
+{{- end -}}
```

Comment:
Remove this helper, it is not used

Reply:
Removed [9a7e6bf](https://github.com/bitnami/charts/pull/13480/commits/9a7e6bf748e2aad0f3bfb9a74e1603bdef89a3f5)